### DISCUSSION TITLE
Python/%E7%B3%BB%E5%88%97%E6%96%87%E7%AB%A0/23%E7%A7%8D%E8%AE%BE%E8%AE%A1%E6%A8%A1%E5%BC%8F/%E5%B7%A5%E5%8E%82%E6%96%B9%E6%B3%95%E6%A8%A1%E5%BC%8F%EF%BC%9A%E8%A7%A3%E9%94%81%E7%81%B5%E6%B4%BB%E7%9A%84%E5%AF%B9%E8%B1%A1%E5%88%9B%E5%BB%BA%E7%AD%96%E7%95%A5

### DISCUSSION THREAD

**Discussion by giscus** [cpython666/cpython666.github.io#5] [Announcements]
# Python/%E7%B3%BB%E5%88%97%E6%96%87%E7%AB%A0/23%E7%A7%8D%E8%AE%BE%E8%AE%A1%E6%A8%A1%E5%BC%8F/%E5%B7%A5%E5%8E%82%E6%96%B9%E6%B3%95%E6%A8%A1%E5%BC%8F%EF%BC%9A%E8%A7%A3%E9%94%81%E7%81%B5%E6%B4%BB%E7%9A%84%E5%AF%B9%E8%B1%A1%E5%88%9B%E5%BB%BA%E7%AD%96%E7%95%A5

Python斗罗的代码笔记

https://cpython666.github.io/Python/%E7%B3%BB%E5%88%97%E6%96%87%E7%AB%A0/23%E7%A7%8D%E8%AE%BE%E8%AE%A1%E6%A8%A1%E5%BC%8F/%E5%B7%A5%E5%8E%82%E6%96%B9%E6%B3%95%E6%A8%A1%E5%BC%8F%EF%BC%9A%E8%A7%A3%E9%94%81%E7%81%B5%E6%B4%BB%E7%9A%84%E5%AF%B9%E8%B1%A1%E5%88%9B%E5%BB%BA%E7%AD%96%E7%95%A5.html

**Comment by Chendx73** [comment]
作者你好，你这个工厂方法模式的代码示例和简单工厂模式的区别是啥？（初学者，可能是我没弄懂）因为“工厂方法模式则是通过多个方法或类来创建对象，每个方法或类负责创建一个具体类型的对象”在代码里好像没有体现

**Reply by cpython666** [reply]
好像没有简单工厂模式，应该是当时写错了，可以理解例子代码中他这种行为就行了。文章当时多写了，不用扣太细节。
